impy/common.py

```diff
@@ -335,6 +413,11 @@ def __init__(self, interaction_model_def, settings_dict=dict(), **kwargs):
 
         # FORTRAN LUN that keeps logfile handle
         self.output_lun = None
+        
+        # Number of events to generate
+        self.nevents = None
+        # Normalization 1/nevents for histograms
+        self.norm = None
```

Comment:
I recommend to remove `norm` or make it a property. Keeping values in sync by hand is a maintenance nightmare. 

If `norm` only exists to calculating the histogram in the way it is described in the PR, then `norm` should be removed. It is both inefficient and inaccurate to calculate a histogram in this way. I will explain this elsewhere.

Reply:
Ok, it will be removed. Also, nevents could be changed to give the number of generated events and not the number of events passed to generator as it is now.

Reply:
I think that would be a very important change indeed! I was not aware that the two numbers can be different.